win32ss/gdi/ntgdi/freetype.c

```diff
@@ -988,6 +1018,7 @@ IntGdiLoadFontsFromMemory(PGDI_LOAD_FONT pLoadFont,
     if (RtlEqualUnicodeString(&Entry->FaceName, &g_MarlettW, TRUE))
     {
         FontGDI->charset = SYMBOL_CHARSET;
+        FontGDI->codepage = IntCodePageFromCharSet(SYMBOL_CHARSET);
```

Comment:
Is SYMBOL_CHARSET also equal to FT_WinFNT_ID_SYMBOL ?

Reply:
Grep'ed.

```cpp
// <wingdi.h>
#define SYMBOL_CHARSET 2

// <ftwinfnt.h>
#define FT_WinFNT_ID_SYMBOL    2
```